action.yml

```diff
@@ -217,6 +259,13 @@ runs:
         CA_GH_ACTION_IMAGE_TO_BUILD="${{ inputs.acrName }}.azurecr.io/github-action/container-app:${{ github.run_id }}.${{ github.run_attempt }}"
         echo "CA_GH_ACTION_IMAGE_TO_BUILD=${CA_GH_ACTION_IMAGE_TO_BUILD}" >> $GITHUB_ENV
 
+    - name: Determine name of image to build if not provided
+      if: ${{ inputs.registryUrl != '' && inputs.imageToBuild == '' && env.CA_GH_ACTION_ONLY_YAML != 'true' }}
+      shell: bash
+      run: |
+        CA_GH_ACTION_IMAGE_TO_BUILD="${{ inputs.registryUrl }}/github-action/container-app:${{ github.run_id }}.${{ github.run_attempt }}"
+        echo "CA_GH_ACTION_IMAGE_TO_BUILD=${CA_GH_ACTION_IMAGE_TO_BUILD}" >> $GITHUB_ENV    
+
```

Comment:
Let's consolidate these two steps together since we have the `CA_GH_ACTION_REGISTRY_URL` environment variable to use:

```yaml
- name: Determine name of image to build if not provided
  if: ${{ env.CA_GH_ACTION_REGISTRY_URL != '' && inputs.imageToBuild == '' && env.CA_GH_ACTION_ONLY_YAML != 'true' }}
```